src/main/java/com/beerjournal/breweriana/item/rating/persistence/RatingRepository.java

```diff
@@ -61,6 +62,6 @@ public Rating delete(ObjectId ratingId) {
 
     private void notifyUpdate(ObjectId itemId) {
         Set<Rating> itemRatings = ratingCrudRepository.findAllByItemId(itemId);
-        ratingUpdateListeners.forEach(listener -> listener.onUpdate(ItemRatings.of(itemId, itemRatings)));
+        ratingUpdateListeners.forEach(listener -> listener.onInsert(ItemRatings.of(itemId, itemRatings)));
```

Comment:
This is not entirely true as we are _inserting_ single rating, but we are thus _updating_ a collection of given item ratings, and that's what listener reacts on - it listens to collection update (so - rating insert/delete).